code/datums/ruins/space.dm

```diff
@@ -258,6 +258,15 @@
 	always_place = TRUE
 	cost = 0
 
+/datum/map_template/ruin/space/syndicate_space_base
+	name = "Syndicate Space Base"
+	id = "syndie-space-base"
+	description = "A secret base researching illegal bioweapons, it is closely guarded by an elite team of syndicate agents."
+	suffix = "syndie_space_base.dmm"
+	cost = 0
+	always_place = TRUE
```

Comment:
Not sure if making this `always_place` is the best idea

Reply:
There are a couple of reasons for this.
The first is that I want people to play it a lot over the next few weeks so I get another burst of feedback and can consider any tweaks/improvements quickly. Instead of having the feedback strung out over months.
The second is that it provides a bunch of isolated off-station spawns for ghosts, and I feel we often have plenty of ghosts who'd like to make use of this.
If you really want it changed to use the same spawn weight as old lavabase (cost = 20) I'll do it, but personally I'd like to see it get used extensively for at least a few weeks as a form of stress test. I've already uploaded a variant of this map into a bunch of rounds so its already been tested quite a bit, but a longer lasting intense testing period can't hurt.

Reply:
couldn't that be a config setting instead?